### PR TITLE
MN-398,315: monitoring scripts starting grafana and prometheus

### PR DESCRIPTION
**Pull request by sansaian**
add monitoring script for starting grafana and prometheus
https://insolar.atlassian.net/browse/MN-315
https://insolar.atlassian.net/browse/MN-398

### CODE REVIEW

File: component/storer.go [hunk 1]
```diff
@@ -184,7 +184,7 @@ func makeStorer(
 				}
 				err := addresses.Update(wasting)
 				if err != nil {
-					return errors.Wrap(err, "failed to insert wasting")
+					return errors.Wrap(err, "failed to insert vasting")
```
Comment (megge-dream) [inline component/storer.go:187]: vasting ?

Comment (SergeyRadist) [inline component/storer.go:187]: I think it was "vesting" ? Maybe we should rename it everywhere

File: scripts/monitor/monitor.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+set +x
+
+export PROMETHEUS_CONFIG_DIR=../../${LAUNCHNET_MONITOR_DIR}prometheus/
+
+if [[ $# -lt 1 ]]; then
+# 1) if started without params  pretend to be clever and do what is expected:
+# * prepare configuration
+# * shutdown ans start all monitoring services
+# * wait until Jaeger starts (we want to be sure tracer works before start benchmark)
```
Comment (SergeyRadist) [inline scripts/monitor/monitor.sh:23]: remove jaeger

File: component/storer.go (hunk 1 above)
Comment (MikhailPopov) [reply component/storer.go:187]: vesting

Comment (MikhailPopov) [reply component/storer.go:187]: +

File: scripts/monitor/monitor.sh (hunk 2 above)
Comment (MikhailPopov) [inline scripts/monitor/monitor.sh:23]: Is Jaeger used here? Maybe you should remove the line?

File: component/storer.go (hunk 1 above)
Comment (sansaian) [reply component/storer.go:187]: Yep. I think yes. I get from this https://cvj.ch/en/glossary/vesting/

File: scripts/monitor/monitor.sh (hunk 2 above)
Comment (sansaian) [reply scripts/monitor/monitor.sh]: No.remove this line

Comment (sansaian) [reply scripts/monitor/monitor.sh]: done

File: component/storer.go (hunk 1 above)
Comment (sansaian) [reply component/storer.go]: fix it

Comment (sansaian) [reply component/storer.go]: fix it on vesting

**Pull request merged**